code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -295,6 +297,7 @@ var/cat_number = 0
 
 /mob/living/simple_animal/cat/runtime/New(loc)
 	..(loc)
+	stats.addPerk(PERK_TERRIBLE_FATE)
```

Comment:
Funny and I like it, but should probably be added to the cl.

Reply:
Do perks even work on non-humans?